roles/openshift_project_request_template/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- when: openshift_project_request_template_manage
+  block:
+    - name: Create temp file for template
+      command: mktemp /tmp/openshift-ansible-XXXXXX.yaml
+      register: mktemp
+      changed_when: False
+
+    - name: Generate default project template
+      shell: |
```

Comment:
Let's not use shell here.  We can use command and register the output.

Then use copy module like this: https://stackoverflow.com/questions/26638180/write-variable-to-a-file-in-ansible/26640778#26640778

Reply:
Switched to a combination of `command` and `copy`.